src/realm/sync/instructions.hpp

```diff
@@ -185,6 +199,9 @@ struct Payload {
     /// Note: For Mixed columns (including typed links), no separate value is required, because the
     /// instruction set encodes the type of each value in the instruction.
     enum class Type : int8_t {
+        // Special value indicating that a list should be created at the position.
+        List = -5,
```

Comment:
Please update the comment at line 194. Do we also need an `EraseList` option?

Reply:
Comment updated. We just use the normal Instruction::ArrayErase instruction.